packages/web/src/common/store/profile/fetchProfileCollectionsSaga.ts

```diff
@@ -35,15 +40,30 @@ function* fetchProfileCollectionsAsync(
   }
 
   const { user_id, _collectionIds } = user
-  const audiusBackendInstance = yield* getContext('audiusBackendInstance')
-
-  try {
-    const latestCollections = yield* call(
-      audiusBackendInstance.getPlaylists,
-      user_id,
-      null,
-      false
+  function* getPlaylists() {
+    const { data } = yield* call(
+      [sdk.full.users, sdk.full.users.getPlaylistsByUser],
+      {
+        id: Id.parse(user_id)
```

Comment:
This one I _think_ you're safe not passing `userId` since the original request was passing `withUsers=false`, but I'm not sure if there's any other customized information on playlists that we'd be losing.